pydantic/v1/datetime_parse.py

```diff
@@ -147,7 +147,7 @@ def parse_time(value: Union[time, StrBytesIntFloat]) -> time:
     number = get_numeric(value, 'time')
     if number is not None:
         if number >= 86400:
-            # doesn't make sense since the time loop back around to 0
+            # doesn't make sense since the time time loop back around to 0
```

Comment:
typo

Reply:
We need to fix it in v1 first:)

Reply:
It got merged. we need to fix it in `v1`